### DISCUSSION TITLE
Problem with 10K probe and ADS1115

### DISCUSSION THREAD

**Discussion by Crewski** [tagyoureit/nodejs-poolController#750] [Nixie]
I'm trying to switch over to using some 10K thermistors (the ones for pools).  I have a ADS1115 hat for my raspberry pi.  The thermistors are hooked up as followed:
GND - both thermistors have one leg connected to GND
AN0 - pool thermistor leg plus 10K resistor
AN1 - air thermistor leg plus 10K resistor
3.3v - both 10K resistors are terminated here.

I'm trying to test this on my test setup....which is inside and the probe are right next to each other in my house.  In REM, I keep getting a `error: ADS1115 Read Register: , Remote I/O error.`  The voltage readings for the 2 probes are very different even though they are side by side in the same room.  Any ideas?  I don't have any experience with thermistors or ADS1115.

The items I bought are:
10k probes - https://www.amazon.com/dp/B07NWK6GJ4
ADS1115 - https://www.amazon.com/dp/B07PNMXR8B

I currently only have REM installed and running for this test.

![image](https://user-images.githubusercontent.com/8012153/235775840-fbfe3ed8-4c57-489f-b25c-5c3e570960f3.png)
![image](https://user-images.githubusercontent.com/8012153/235775941-132cb390-3ba1-49fa-bad7-d0407ef6e13f.png)

**Comment by Crewski** [comment]
Figured out the error of them reading different.  There is an onboard NTC with a jumper to A01 that needed to be removed.

**Reply by rstrouse** [reply]
Yeah some of those ADS boards also have some weird setups for differential reads.  For a 10k temp probe you want to use single ended readings.   I have included some documentation for setting these up below.
https://github.com/rstrouse/relayEquipmentManager/wiki/Temperature-Sensors
